rocketpy/rocket/aero_surface.py

```diff
@@ -167,7 +168,7 @@ def __init__(
             Nose cone length. Has units of length and must be given in meters.
         kind : string
             Nose cone kind. Can be "conical", "ogive", "elliptical", "tangent",
-            "von karman", "parabolic" or "lvhaack".
+            "von karman", "parabolic", "powerseries" or "lvhaack".
```

Comment:
In this section, I would describe it a bit more... 
Maybe adding something like:  If "powerseries" is used, the "power" argument must be assigned to a value between 0 and 1.

```
            "von karman", "parabolic", "powerseries" or "lvhaack".
```